app/views/comments/create.html.erb

```diff
@@ -0,0 +1,12 @@
+<!-- Creating error message displaying the data that failed -->
+<% if @comment.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@comment.errors.count, "error") %> prohibited this comment from being saved:</h2>
+
+    <ul>
+    <% @comment.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
```

Comment:
- [ ] It is required to have a form where the current user can add comments to a post. That is not implemented yet. Kindly add that too.

Reply:
Hi, 
I wanted you to know that the user can create a comment by going to a specific post as you can see in the picture below
![Screenshot 2023-08-23 021556](https://github.com/IsmailChangezi/blogsphere/assets/96125730/b4b79787-7691-475d-9045-d5ccb4319185)